gazebo/.gitignore

```diff
@@ -11,3 +11,4 @@
 
 pkg/
 src/
+gazebo
```

Comment:
Why is this needed?

I actually prefer to keep to have no `.gitignore` file for PKGBUILDs so that I can remove build artifacts easily with `git clean -fd`.
This doesn't have to be addressed for this PR.

Reply:
I will remove that line, and we can talk about the pros and cons of using the gitignore file for build files in a separate issue. I actually prefer having a gitignore for build files. And, the gazebo PKGBUILD folder already had a gitignore, so I'm sure I'm not alone in that camp.

Reply:
I'm with @bobosito000 in that I prefer having ``.gitignore`` files. But yes, let's talk about this elsewhere.